shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -540,8 +549,12 @@ void clearTextInputClient() {
       // InputConnection is managed by the TextInputPlugin, and events are forwarded to the Flutter
       // framework.
       FRAMEWORK_CLIENT,
-      // InputConnection is managed by an embedded platform view.
-      PLATFORM_VIEW
+      // InputConnection is managed by an embedded platform view that is backed by a virtual
+      // display.
+      VD_PLATFORM_VIEW,
+      // InputConnection is managed by an embedded platform view that is embeded in the Android view
+      // hierarchy.
+      HC_PLATFORM_VIEW,
```

Comment:
Is `HC` meant to stand for "hierarchy"? Not sure if this is a standard abbreviation, but it was not intuitive for me. It may be clearer to just write it out.

Reply:
I clarified it in the comment above.